test/functionalTest/cases/3177_updateContext_APPEND_CPr_Registration/updateContext_APPEND_CPr_Reg.test

```diff
@@ -0,0 +1,212 @@
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U

```

Comment:
2014 -> 2018

Reply:
Hi @fgalan Sir,
I have applied all the corrections on .test file, Please review it and guide me further.
Thank You !

Reply:
Fixed in 589148e